<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutputPreferences</class>
 <widget class="QWidget" name="OutputPreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>269</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="fontGroup">
     <property name="title">
      <string>&amp;Font</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="fontLabel">
          <property name="text">
           <string>F&amp;amily:</string>
          </property>
          <property name="buddy">
           <cstring>fontSelect</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFontComboBox" name="fontSelect"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="sizeLabel">
          <property name="text">
           <string>&amp;Size:</string>
          </property>
          <property name="buddy">
           <cstring>sizeSpin</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sizeSpin"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="editorBox">
        <property name="text">
         <string>Use &amp;editor settings</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="colourGroup">
     <property name="title">
      <string>&amp;Colour</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="foreLabel">
        <property name="text">
         <string>F&amp;oreground:</string>
        </property>
        <property name="buddy">
         <cstring>foreButton</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QtColorPicker" name="foreButton">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="backLabel">
        <property name="text">
         <string>&amp;Background:</string>
        </property>
        <property name="buddy">
         <cstring>backButton</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QtColorPicker" name="backButton">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="syscolBox">
        <property name="text">
         <string>&amp;Use system colours</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtColorPicker</class>
   <extends>QPushButton</extends>
   <header location="global">qtcolorpicker.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>fontSelect</tabstop>
  <tabstop>sizeSpin</tabstop>
  <tabstop>editorBox</tabstop>
  <tabstop>foreButton</tabstop>
  <tabstop>backButton</tabstop>
  <tabstop>syscolBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
